tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "baseUrl": "src/scripts",
+    "jsx": "react",
+    "lib": [
+      "DOM",
+      "ES6",
+      "ScriptHost",
+      "DOM.Iterable"
+    ],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "paths": {},
+    "sourceMap": true,
+    "strict": true,
+    "target": "es5"
+  }
+}
```

Comment:
What about decorators? I like them in some cases... `"experimentalDecorators": true`

Reply:
With `"allowSyntheticDefaultImports": true` you can write `import React from 'react'` instead of `import * as React from 'react'`.

Reply:
Done (both).